web/src/modules/i18n/containers/I18nProvider.js

```diff
@@ -47,37 +47,54 @@ export class I18nProvider extends React.Component<PropsType, StateType> {
       interpolation: {
         escapeValue: false // Escaping is not needed for react apps: https://github.com/i18next/react-i18next/issues/277
       },
-      debug: buildConfig().featureFlags.developerFriendly
+      debug: buildConfig().featureFlags.developerFriendly,
+      detection: {
+        order: ['localStorage', 'navigator']
+      }
     })
+
     this.state = {
-      language: DEFAULT_LANGUAGE,
+      language: this.i18n.language,
       fonts: I18nProvider.getSelectedFonts(DEFAULT_LANGUAGE),
```

Comment:
shouldn't this be
```suggestion
      fonts: I18nProvider.getSelectedFonts(this.i18n.language),
```

Reply:
Good catch, probably overwritten by the other pr thought!